src/__init__.py

```diff
@@ -221,11 +221,19 @@ def run_migrate(clients: Clients, user_inputs: UserInputs) -> MigrateOutputs | N
 
     # Check difference with main
     changes = recreate_docs(clients, DOCUMENTATION_TAG)
+    # Check whether there are still changes after switching to the base branch
+    if changes:
+        changes = clients.repository.is_dirty(user_inputs.base_branch)
+
+        # Move the tag if there are no changes
+        if not changes:
+            with clients.repository.with_branch(user_inputs.base_branch) as repo:
+                main_hash = repo.current_commit
+            clients.repository.tag_commit(DOCUMENTATION_TAG, main_hash)
```

Comment:
**suggestion** maybe it is not needed to defined `main_hash`. Would this work?

```
with clients.repository.with_branch(user_inputs.base_branch) as repo:
    clients.repository.tag_commit(DOCUMENTATION_TAG, repo.current_commit)
```

Reply:
I think it is required because otherwise the branch at this point isn't the base branch. I tried removing it and the wrong commit SHA is tagged

Reply:
On line 218 it is done in the same way